tests/connectInstance_test.py

```diff
@@ -0,0 +1,78 @@
+#  Copyright 2023 Adobe. All rights reserved.
+#  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License. You may obtain a copy
+#  of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under
+#  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+#  OF ANY KIND, either express or implied. See the License for the specific language
+#  governing permissions and limitations under the License.
+
+import unittest
+from aepp import configs,ConnectObject
+from unittest.mock import patch, MagicMock, Mock, ANY
+
+@patch("aepp.configs.importConfigFile")
+def test_importConfigFile_Oauth(mock_data):
+    mock_data.return_value = {
```

Comment:
I'm not sure we need this test, from what I see its just saying that mock_data is a dict, is there a case ever where it wouldnt be a dict?

Reply:
You can just define mock_data as a fixture that can be leveraged in the methods below